flag/service/kubernetes/tls/tls.go

```diff
@@ -0,0 +1,9 @@
+package tls
+
+// TLS is a data structure for Kubernetes TLS configuration with command line
+// flags
+type TLS struct {
+	CAFile  string
+	CrtFile string
+	KeyFile string
+}
```

Comment:
The actual go project structure could have its separate PR.

Reply:
When submitting the Go project structure, shall I include or exclude `dep` files (`Gopkg.*`)?

Reply:
The dep files have always have to be there, otherwise the code will not build.